CHANGELOG.md

```diff
@@ -1,5 +1,11 @@
 # Changelog
 
+## Unreleased
+
+### Fixes
+
+* Updates uses of "whitelist" to "allowlist" to follow project practices.
```

Comment:
does this need a changelog entry? It'll mostly be seen in docs.

Reply:
Can't hurt. I like that we're putting it out there.